WISE-Community/WISE
# Added content to site footer

To test:
- Load the site and review the footer content. Make sure all the links work and the layout is responsive.

Closes #1544.
